security-privacy-self-assessment.md

```diff
@@ -0,0 +1,84 @@
+# Security and Privacy Self-Review Questionnaire for DeviceOrientation Event
+
+[Self-Review Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+responses for the [DeviceOrientation Event](https://www.w3.org/TR/orientation-event/) specification.
+
+Related Self-Review Questionnaires:
+- [Generic Sensor API](https://github.com/w3c/sensors/blob/main/security-questionnaire.md)
+- [Orientation Sensor API](https://github.com/w3c/orientation-sensor/blob/main/security-questionnaire.md)
+
+### 2.1 What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+Information about the physical orientation and movement of the hosting device. See [use cases](https://www.w3.org/TR/orientation-event/#use-cases).
+
+### 2.2 Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+This specification allows developers to request access to only the set of sensor data necessary, either a "device motion" event for motion-controlled applications or a "device orientation" event for orientation control. Since absolute orientation exposes more information, a "relative orientation" option is available if the compass heading is unnecessary.
+
+The data precision is normatively limited to resist fingerprinting (see 2.6).
+
+### 2.3 How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+
+This specification does not deal with PII directly, however identifiable information such as gait can be determined from monitoring sensor readings over time. The specification requires users to give express permission for the user agent to provide device motion and/or orientation data.
+
+### 2.4 How do the features in your specification deal with sensitive information?
+
+See 2.3.
+
+### 2.5 Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+No.
```

Comment:
```suggestion
This specification defines new [permissions](https://www.w3.org/TR/orientation-event/#permissions-api-integration) for which user agents may persist the user's decision to grant access to a site between browsing sessions. Otherwise, sensors are completely stateless. 
```